bindings/core/src/method_handler/wallet.rs

```diff
@@ -399,8 +396,13 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
             let transaction = wallet.send_outputs(outputs, options).await?;
             Response::SentTransaction(TransactionWithMetadataDto::from(&transaction))
         }
+        // TODO #1934: create issue
```

Comment:
TODO

Reply:
Are you actually sure that we want this?

Reply:
No I don't think there's a definitive reason for adding this. I removed this commented code. We could still add an issue if you think it was a valid idea.

Reply:
Currently I don't see when one would need to set it later, so personally I wouldn't open an issue